### DISCUSSION TITLE
smtp connect() failed. https://github.com/phpmailer/phpmailer/wiki/troubleshooting

### DISCUSSION THREAD

**Discussion by Manisha221197** [PHPMailer/PHPMailer#2469] [Q&A]
Hello all,
I'm getting this error on live server (namecheap)
The phpmailer working on localhost but when uploading into server and booking any service on our website we are getting the smtp error.
Please help us to solve this error.

**Comment by Synchro** [comment]
This is a very common issue usually caused by your hosting provider blocking outbound SMTP. Please read the troubleshooting guide for how to diagnose it. You may need to ask your hosting provider to enable outbound SMTP, or you will need to find a better hosting provider.
